site.step-80-monitoring.yml

```diff
@@ -1,7 +1,13 @@
+- import_playbook: site.common.group-current-hosts.yml
+
 - hosts: localhost
   connection: local
   roles:
     - role: cs.aws-region-facts
     - role: cs.aws-logs-slack
       when: aws_logs_slack_notifications
     - role: cs.aws-logs-retention
+- hosts: app:&current
+  roles:
+    - role: cs.warmup
```

Comment:
If we need more EC2 at start it's better to wait with monitoring when all will be up.

Reply:
The main reason to introduce this was to prevent alerts just after deployment.
We need at leads one node that's able to accept traffic to no longer trigger them.
if for some reason more app nodes is terminated (eg. old node from asg not yet stopped - we have cases like this from time to time) this will use not functional node and  never pass no matter how many retries we give it.
If that would be real issue that should be investigated, we have better monitoring that would report that and ASAIK most of those caches are warmed into shared storage (redis/efs) so when single node pass, all of them should.
First node should be relatively safe as it would most likely fail first at post-deployment step.
Regardless better to reduce this probability by sticking to single node.

Reply:
Done